### PR TITLE
feat(mobile): dynamic text welcome screen bb-305

### PR DESCRIPTION
**Pull request by Olha-Katran**
Add dynamic text for welcome screen which appears line by line 

Video:

https://github.com/user-attachments/assets/a727edf9-3b5d-49fc-9a49-9c9040292faf

### CODE REVIEW

File: apps/mobile/src/libs/helpers/create-animated-text/create-animated-text.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	initDelay: number = TextAnimationConfig.INIT_DELAY,
+): React.ReactNode => {
+	const words = text.split(" ");
+
+	return words.map((word, index) => {
+		const animatedOpacity = new Animated.Value(
+			TextAnimationConfig.INITIAL_OPACITY,
+		);
+
+		Animated.timing(animatedOpacity, {
```
Comment (revenets) [inline apps/mobile/src/libs/helpers/create-animated-text/create-animated-text.tsx:18]: Use `reanimated` library for animation

**Review by revenets — CHANGES_REQUESTED** [review]

File: apps/mobile/src/libs/helpers/create-animated-text/create-animated-text.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Animated } from "react-native";
+
+import { Text } from "~/libs/components/components";
+import { BaseColor, TextAnimationConfig } from "~/libs/enums/enums";
+
+const createAnimatedText = (
```
Comment (fshabanov) [inline apps/mobile/src/libs/helpers/create-animated-text/create-animated-text.tsx:7]: Let's move to a custom hook, as we did on web

File: apps/mobile/src/screens/welcome/libs/constants/constants.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { AnalyzingText } from "./analyzing-text.constants";
```
Comment (fshabanov) [inline apps/mobile/src/screens/welcome/libs/constants/constants.ts:1]:
```suggestion
export { AnalyzingText } from "./analyzing-text.constant";
```

File: apps/mobile/src/screens/welcome/welcome.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useCallback, useEffect, useState } from "react";
```
Comment (fshabanov) [inline apps/mobile/src/screens/welcome/welcome.tsx:1]: Should import hooks from ~/libs/hooks/hooks

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**